web/docs/careers/sre.mdx

```diff
@@ -0,0 +1,38 @@
+---
+id: careers-sre
+slug: sre
+title: Cloud Platform Engineers and SREs.
+description: Supabase is hiring Cloud Platform Engineers and SREs
+---
+
+import Roles from '@site/docs/careers/snippets/roles.mdx'
+import About from '@site/docs/careers/snippets/about.mdx'
+import Team from '@site/docs/careers/snippets/team.mdx'
+import Apply from '@site/docs/careers/snippets/apply.mdx'
+
+
+## About the role
+
+Supabase is hiring Cloud Platform Engineers and Site Reliability Engineers.
+
+### Responsibilities
+
+- Help build the Supabase Cloud offering.
```

Comment:
You will be in charge of improving our platform uptime, security and improving our cloud architecture. 

- Help build the Supabase Cloud offering 
- Improve our system architecture to reduce costs while balancing security and performance
- Work on designing and tracking metrics for platform uptime
- Increase observability into our system by capturing relevant metrics and logs
- Implement and maintain intrusion detection, automated remediation and patch management systems
- Design CI/CD systems to speed up deploys with proper change and release management processes
- Work on our SOC2 and GDPR compliance initiatives

Reply:
Ja aber wie